test/rule-matches/color-contrast-matches.js

```diff
@@ -337,6 +337,26 @@ describe('color-contrast-matches', function() {
 		assert.isFalse(rule.matches(target, axe.utils.getNodeFromTree(target)));
 	});
 
+	it("should match when at least one input is not disabled - aria-labelledby", function() {
+		fixture.innerHTML =
+			'<div id="t1">Test</div>' +
+			'<input type="text" aria-labelledby="t1" disabled />' +
+			'<input type="text" aria-labelledby="t1" />';
+		var target = fixture.querySelector('div');
+		axe.testUtils.flatTreeSetup(fixture);
+		assert.isTrue(rule.matches(target, axe.utils.getNodeFromTree(target)));
+	});
+
+	it("should match when inputs are disabled by an ancestor - aria-labelledby", function() {
```

Comment:
Shouldn't this example not match as every `aria-lablledby` input is disabled by the parent fieldset so the `div` label should be treated as disabled?

Reply:
Jup, it should, and it would have if I hadn't messed up the test in two places :P